randovania/layout/preset_migration.py

```diff
@@ -977,6 +977,16 @@ def _migrate_v62(preset: dict) -> dict:
     return preset
 
 
+def _migrate_v63(preset: dict) -> dict:
+    if preset["game"] == "prime1":
+        if preset["configuration"]["qol_cutscenes"] in ["original", "skippable"]:
+            preset["configuration"]["qol_cutscenes"] = "skippable"
+        else:
+            preset["configuration"]["qol_cutscenes"] = "skippablecompetitive"
```

Comment:
Is this the enum name? ugh. Could've been `skippable-competitive` but I see this is an existing value.

Reply:
This value is used by the patcher. Ideally it's `SkippableCompetitive` but I didn't want to rename existing to make consistent

Reply:
At a later date, I will have a PR that updates everything used by the prime exporter to conform to the new schema

Reply:
I'll make that separation in the schema conformity PR then